frontend/src/component/layout/MainLayout/DraftBanner/DraftBanner.tsx

```diff
@@ -94,30 +123,31 @@ export const DraftBanner: VFC<IDraftBannerProps> = ({ project }) => {
     const [isSidebarOpen, setIsSidebarOpen] = useState(false);
     const { data, loading } = usePendingChangeRequests(project);
 
+    const drafts = useMemo(
+        () =>
+            data?.filter(changeRequest =>
+                ['Draft', 'In review', 'Approved'].includes(changeRequest.state)
```

Comment:
why is approved and in review added to drafts variable? it doesn't make sense to me